packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -3,13 +3,17 @@ import {Observable} from 'rxjs'
 import chalk from 'chalk'
 import promptForDatasetName from '../../actions/dataset/datasetNamePrompt'
 import validateDatasetName from '../../actions/dataset/validateDatasetName'
+import listDatasetCopyJobs from '../../actions/dataset/listDatasetCopyJobs'
 import debug from '../../debug'
 
 const helpText = `
 Options
   --detach Start the copy without waiting for it to finish
   --attach <job-id> Attach to the running copy process to show progress
   --skip-history Don't preserve document history on copy
+  --list Lists all dataset copy jobs corresponding to a certain criteria.
+  --offset Start position in the list of jobs. Default 0.
+  --limit Maximum number of jobs returned. Default 10. Maximum 1000.
```

Comment:
🤔 These flags are only relevant when `--list` is specified. Maybe something like:
```suggestion
  --offset Start position in the list of jobs. Default 0. With --list.
  --limit Maximum number of jobs returned. Default 10. Maximum 1000. With --list.
```
For clarity?